syntax = "proto3";

package Ye_Interface;

message InterfaceRequest
{
    bytes type = 1;
    bytes request_msg = 2;
}

message InterfaceResponse
{
    bytes type = 1;
    bytes response_msg = 2;
}

message ChatMsg
{
    int32 to_user_id = 1;
    int32 from_user_id = 2;
    bytes msg = 3;
    bytes time = 4;
    bytes user_name = 5;
}

message GroupMsg
{
    int32 group_id = 1;
    int32 from_user_id = 2;
    bytes user_name = 3;
    bytes msg = 4;
    bytes time = 5;
}